R/strategies.R

```diff
@@ -1,16 +1,27 @@
+
+
+
 #' Compute covariance matrix for some reference-based methods (JR, CIR)
 #'
 #' @description
 #' Adapt covariance matrix in reference-based methods. Used for Copy Increments in
 #' Reference (CIR) and Jump To Reference (JTR) methods, to adapt the covariance matrix
-#' to different pre-deviation and post deviation covariance structures.
+#' to different pre-deviation and post deviation covariance structures. See Carpenter
+#' et al. (2013)
 #'
-#' @param sigma_group TODO
-#' @param sigma_ref TODO
-#' @param index_mar TODO
+#' @param sigma_group the covariance matrix with dimensions equal to `index_mar` for
+#' the subjects original group
```

Comment:
"the covariance matrix with dimensions equal to the length of `index_mar` for the subjects treatment group"

Reply:
But its not necessarily a treatment group is it ?

Reply:
You're right :).